jobs/metricsforwarder/spec

```diff
@@ -70,19 +71,19 @@ properties:
     default: false
 
   autoscaler.metricsforwarder.cache_ttl:
-    description: ""
+    description: "custom metric credential cache ttl"
     default: 900s
   autoscaler.metricsforwarder.cache_cleanup_interval:
-    description: ""
+    description: "interval to clean up expired item in ache"
     default: 6h
   autoscaler.metricsforwarder.policy_poller_interval:
-    description: ""
+    description: "interval to fresh local app info from policy database"
```

Comment:
The time interval to refresh cached application information from policy database